domains/rntxpurval.is-a-fullstack.dev.json

```diff
@@ -1,15 +1,16 @@
 {
     "description": "For Hosting My projects",
     "domain": "is-a-fullstack.dev",
-    "subdomain": "rntxpurval",
+    "subdomain": "tomato.rntxpurval",
 
     "owner": {
         "repo": "https://github.com/purval26/food-del",
         "email": "purvalradadiya@gmail.com"
     },
 
     "record": {
-        "CNAME": "tomato.rntxpurval.is-a-fullstack.dev"
+        "CNAME": "food-frontend-5nj0.onrender.com",
+        "CNAME": "www.food-frontend-5nj0.onrender.com"
```

Comment:
Double CNAMEs is not possible. CNAMEs can only be one single hostname.

Reply:
all issues fixed
@DEV-DIBSTER